### PR TITLE
Title helper directive

### PR DESCRIPTION
**Pull request by Jade-GG**

### CODE REVIEW

File: resources/views/layouts/title.blade.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<title>
+    @hasSection('title')
+        {{ Rapidez::config('design/head/title_prefix') }}
+        @yield('title')
+        {{ Rapidez::config('design/head/title_suffix') }}
+    @else
+        {{ Rapidez::config('design/head/default_title') }}
```
Comment (indykoning) [inline resources/views/layouts/title.blade.php:7]: just in case, this should not be prefixed and suffixed right?

**Comment by royduin** [comment]
We don't need this on 0.x? And we need to update this table: https://docs.rapidez.io/1.x/configuration.html#magento

File: resources/views/layouts/title.blade.php (hunk 1 above)
Comment (Jade-GG) [reply resources/views/layouts/title.blade.php]: Correct, the prefix and suffix only come into play when you have a title set. This follows the standard Magento behavior.

**Comment by Jade-GG** [comment]
https://github.com/rapidez/core/pull/273
https://github.com/rapidez/docs/pull/37

😇

**Pull request merged**